tests/unit/backend/fixtures/test_db.py

```diff
@@ -199,6 +261,122 @@ def _create_test_dataset_artifacts(self):
         self.session.add(dataset_artifact)
         self.session.commit()
 
+        # Revision 1
+
+        self.session.add(
```

Comment:
@MDunitz I added all the artifact formats for each dataset, since that's required in FE before allowing publishing a revision 😆 PTAL if the format looks good! Thank you!

Reply:
They look good. However I'm a little worried that the update to the explorer to search based on the explorer url will break the functional tests. I think we will need to update the explorer_urls on lines 225 and 205.

Reply:
ahh I see! Great catch 🎣  Do we just change `"test_url"` to a real explorer url or something? E.g., `"https://cellxgene.cziscience.com/e/a1b9c51e-a408-4f7f-bccb-abefe20ae2a5.cxg"`?

Reply:
Will sync with @MDunitz tomorrow morning to figure out the best plan forward to avoid breaking functional tests!